server/pulp/server/webservices/urls.py

```diff
@@ -1,6 +1,7 @@
 from django.conf.urls import patterns, url
 
-from pulp.server.webservices.views.consumer_groups import (ConsumerGroupAssociateActionView,
+from pulp.server.webservices.views.consumer_groups import (ConsumerGroupView,
```

Comment:
I usually alphabetize the imports.so `ConsumerGroupView` would come towards the end.